### ISSUE TITLE
Installer is broken

### ISSUE THREAD

**Issue by skipperguy12** [NJayDevelopment/mongoid_forums#16]
```
$ rails g mongoid_forums:install
What is your user class called? [User] 
What is the current_user helper called in your app? [current_user] 
Defining mongoid_forums_user method inside ApplicationController...
      insert  app/controllers/application_controller.rb
Adding mongoid_forums initializer (config/initializers/mongoid_forums.rb)...
      create  config/initializers/mongoid_forums.rb
(erb):5:in `template': undefined method `per_page' for MongoidForums:Module (NoMethodError)
```

The route is successfully added, however the initializer/mongoid_forums.rb is a blank file. The method is defined exactly how radar/forem does it, what could be the error?

**Comment by skipperguy12** [comment]
Per page method definition: https://github.com/NJayDevelopment/mongoid_forums/blob/master/lib/mongoid_forums.rb#L33

Mattr accessor:
https://github.com/NJayDevelopment/mongoid_forums/blob/master/lib/mongoid_forums.rb#L9

**Comment by AustinLMayes** [comment]
The installer worked for me earlier today, just backtrack and also try bundle installing and recreating the test app environment, could be a weird cacheing issue.

**Comment by skipperguy12** [comment]
@AustinLMayes Did so already, @gcflames5 also couldn't get the installer working without running it a bunch of times (I could do this at all). Could you possibly show me how you're creating the app and installing it?

**Comment by skipperguy12** [comment]
Posted to Stack Overflow: http://stackoverflow.com/questions/29553299/install-generator-wont-see-a-method-i-defined

**Comment by AustinLMayes** [comment]
@skipperguy12 http://hastebin.com/elicatucob.coffee

**Comment by skipperguy12** [comment]
@AustinLMayes I see that you're in test/dummy. I'm trying it on a new app, and it fails. What's different and special about test/dummy?

**Comment by AustinLMayes** [comment]
@skipperguy12 It appears the wrong class is being loaded by Rails. If you paste the class in rails console and run per_page, it returns a value. If not pasted, calling MongoidForums.per_page returns an undefined method error.

**Comment by AustinLMayes** [comment]
@radar any thoughts?

**Comment by skipperguy12** [comment]
@spaceghost found the issue. Turns out when you try requiring mongoid_forums in pry, you'll see that an error involving decorators occurs. The issue is fixed here: https://github.com/parndt/decorators/pull/13

It's because of the way files are required and how load! is called over there.

**Comment by Spaceghost** [comment]
@skipperguy12  [decorators-1.0.5](https://rubygems.org/gems/decorators/versions/1.0.5) is the release with the fix in it, and anything after 2.0.1 also has the fix. Thanks @parndt.
